apps/io-services-cms-webapp/src/reviewer/__tests__/request-review-handler.test.ts

```diff
@@ -105,6 +123,15 @@ const aDbInsertData: ServiceReviewRowDataTable = {
   extra_data: {},
 };
 
+const aRejectDbInsertData: ServiceReviewRowDataTable = {
```

Comment:
Why did we call it `aRejectDbInsertData` when instead the status is `"PENDING"`?

Reply:
i didn't know how to call it since when you reopen a rejected ticket (reject->new) we need to create a pending entry on the postgres db , basically is a new version for a previously rejected ticket